app/src/main/java/com/gh4a/activities/issues_label/IssueLabelListActivity.java

```diff
@@ -214,8 +207,10 @@ private void updateFabVisibility() {
 
     private final class EditActionMode implements ActionMode.Callback {
         private final IssueLabelAdapter.EditableLabel mLabel;
+        private final Activity mActivity;
 
-        public EditActionMode(IssueLabelAdapter.EditableLabel label) {
+        public EditActionMode(Activity activity, IssueLabelAdapter.EditableLabel label) {
+            mActivity = activity;
```

Comment:
Passing activity is unneeded here, the class isn't static, thus the activity can be accessed as IssueLabelListActivity.this.